src/UIComponents/GenerateResults.jsx

```diff
@@ -181,14 +181,23 @@ class GenerateResults extends Component {
           }}
         >
           <div className="ailab-image-hover" style={styles.trainBot}>
-            <img src={aiBotHead} style={styles.trainBotHead} />
-            <img src={aiBotBody} style={styles.trainBotBody} />
+            <img
+              src={aiBotHead}
+              style={styles.trainBotHead}
+              alt="A.I. Bot head"
+            />
+            <img
+              src={aiBotBody}
+              style={styles.trainBotBody}
+              alt="A.I. Bot body"
+            />
```

Comment:
I'm a bit torn about including "Bot" because of our [previous naming discussions](https://codedotorg.slack.com/archives/C016VGH1BT6/p1616203569028600?thread_ts=1615308631.003000&cid=C016VGH1BT6).  Maybe "bot" to be descriptive without formally naming it?